tools/networkmonitor/networkmonitor.nim

```diff
@@ -193,7 +193,33 @@ proc crawlNetwork(node: WakuNode,
     # we dont run ipMajorityLoop
     # we dont run revalidateLoop
 
-    await sleepAsync(crawlInterval)
+    await sleepAsync(crawlInterval.millis)
+
+proc retrieveDynamicBootstrapNodes(dnsDiscovery: bool, dnsDiscoveryUrl: string, dnsDiscoveryNameServers: seq[ValidIpAddress]): Result[seq[RemotePeerInfo], string] =
+  if dnsDiscovery and dnsDiscoveryUrl != "":
+    # DNS discovery
+    debug "Discovering nodes using Waku DNS discovery", url=dnsDiscoveryUrl
+
+    var nameServers: seq[TransportAddress]
+    for ip in dnsDiscoveryNameServers:
+      nameServers.add(initTAddress(ip, Port(53))) # Assume all servers use port 53
+
+    let dnsResolver = DnsResolver.new(nameServers)
+
+    proc resolver(domain: string): Future[string] {.async, gcsafe.} =
+      trace "resolving", domain=domain
+      let resolved = await dnsResolver.resolveTxt(domain)
+      return resolved[0] # Use only first answer
+
+    var wakuDnsDiscovery = WakuDnsDiscovery.init(dnsDiscoveryUrl, resolver)
```

Comment:
Not bit deal but shall use let instead?
```suggestion
    let wakuDnsDiscovery = WakuDnsDiscovery.init(dnsDiscoveryUrl, resolver)
```

Reply:
Actually, this broke the build 🤣 Reverting it!